Specs/DateTime/English/MergedParser.json

```diff
@@ -793,45 +794,26 @@
     ]
   },
   {
-    "Input": "Set an appointment for Easter Sunday",
```

Comment:
Please don't change the previous case. Add the new one as a separate test.

Reply:
Got it, will update shortly.

Reply:
Done